frontend/mgramseva/lib/screeens/Updatepassword.dart

```diff
@@ -330,7 +332,11 @@ class _UpdatePasswordState extends State<UpdatePassword> {
                 radius: Radius.circular(1),
                 enabled: true,
               ),
-              pinLength: 6,
+              onChanged: (String value){
+                  setState(() {
```

Comment:
Why empty setstate is happening

Reply:
For disabling or enabling the submit button

Reply:
Forcing the ui to re-render